### DISCUSSION TITLE
Cannot pass inputs with local actions

### DISCUSSION THREAD

**Discussion by sakebook** [community/community#25867] [Actions]
<p>I tried Docker container actions tutorial.</p>
<p><a href="https://help.github.com/en/articles/creating-a-docker-container-action" rel="nofollow noopener">https://help.github.com/en/articles/creating-a-docker-container-action</a></p>
<p>Cannot work private action.</p>
<p><a href="https://help.github.com/en/articles/creating-a-docker-container-action#example-using-a-private-action" rel="nofollow noopener">https://help.github.com/en/articles/creating-a-docker-container-action#example-using-a-private-action</a></p>
<p>I created simple sample. </p>
<ul>
<li>public.yml</li>
</ul>
<p>  - Use public action.</p>
<ul>
<li>main.yml</li>
</ul>
<p>  - Use private action in <code>.github/actions</code></p>
<ul>
<li>local.yml</li>
</ul>
<p>  - Use private action in root, but cannot pass the inputs</p>
<p><a href="https://github.com/sakebook/hello-world-docker-action" rel="nofollow noopener">https://github.com/sakebook/hello-world-docker-action</a></p>
<p>I think same main.yml and local.yml.</p>
<p>But the result is different.</p>
<ul>
<li>main.yml</li>
</ul>
<p>  - <a href="https://github.com/sakebook/hello-world-docker-action/commit/8f237bd05e82b89525b9c343e17b21ec57037d87/checks?check_suite_id=254478598" rel="nofollow noopener">https://github.com/sakebook/hello-world-docker-action/commit/8f237bd05e82b89525b9c343e17b21ec57037d87/checks?check_suite_id=254478598</a></p>
<ul>
<li>local.yml</li>
</ul>
<p>  - <a href="https://github.com/sakebook/hello-world-docker-action/commit/8f237bd05e82b89525b9c343e17b21ec57037d87/checks?check_suite_id=254478544" rel="nofollow noopener">https://github.com/sakebook/hello-world-docker-action/commit/8f237bd05e82b89525b9c343e17b21ec57037d87/checks?check_suite_id=254478544</a></p>
<p>local is cannnot pass <code>who-to-greet</code></p>
<p>This is a bug? Or specification?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi,</p>
<p>This is due to incorrect action.yml under root directory.</p>
<p>The action metadata file should be named as ‘action.yml’, there is an extra ‘s’ in your file name.</p>
<p>Official doc:</p>
<p><a href="https://help.github.com/en/articles/metadata-syntax-for-github-actions" rel="nofollow noopener">https://help.github.com/en/articles/metadata-syntax-for-github-actions</a></p>

**Comment by sakebook** [comment]
<p>Holy clap!</p>
<p>I appreciate your help. </p>
